cmd/build.go

```diff
@@ -107,23 +117,21 @@ func getLdflags(info ProjectInfo) string {
 				"repoPath": RepoPathFunc,
 				"user":     UserFunc,
 			}
-			ldflags = viper.GetString("build.ldflags")
+			ldflagsTmpl = viper.GetString("build.ldflags")
 		)
 
-		tmpl, err := template.New("ldflags").Funcs(fnMap).Parse(ldflags)
+		tmpl, err := template.New("ldflags").Funcs(fnMap).Parse(ldflagsTmpl)
 		fatalMsg(err, "Failed to parse ldflags text/template")
 
 		err = tmpl.Execute(tmplOutput, info)
 		fatalMsg(err, "Failed to execute ldflags text/template")
 
-		if goos != "darwin" {
-			tmplOutput.WriteString("-extldflags \"-static\"")
-		}
-
-		return tmplOutput.String()
+		ldflags = append(ldflags, tmplOutput.String())
```

Comment:
Is it valid to define ldflags multiple times? E.g. in case someone sets `-s` also in `.promu.yml`? Should check if `-s` and static flags are already present first?